### DISCUSSION TITLE
Option to auto-disconnect other client when a new client connects.

### DISCUSSION THREAD

**Discussion by CubicPants** [LizardByte/community#281] [Sunshine] [answered]
Hi All,

Has this been discussed or suggested before?
From memory, NVIDIA Gamestream worked this way.

I'm finding myself having to VNC to the host to restart Sunshine (prefer not to open web ports).

**Comment by ReenigneArcher** [comment, marked as answer]
set channels = 2 and you can connect from another session... but best to end your first session.

**Reply by CubicPants** [reply]
That works for me. Thanks ReenigneArcher.
